src/components/FilterTodo/FilterTodo.tsx

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/TodoStatus';
+
+export function filterTodos(todos: Todo[], selectedFilter: TodoStatus): Todo[] {
+  return todos.filter((todo) => {
+    switch (selectedFilter) {
+      case TodoStatus.All:
+        return true;
+
```

Comment:
if you have filtering by ALL, you'll jump on default case from this switch that doing actually the same => returning true. So you can just remove this case and `default` will do it :)